### DISCUSSION TITLE
Game not launching after compiling

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2459] [Q&A]
Elaboration:
i type "lime test windows", it goes ok, but at the very last it gives me:
![image](https://user-images.githubusercontent.com/60806448/145566087-952352f5-f57b-498a-af63-9da7ef9bd4a4.png)
(don't mind the russian language)
at this momen the game SHOULD launch, but it does not
BUT! i have to mention, that i still can launch it manually, like any other game

Am I doing smth wrong?
(tho i can share the code if needed)

**Comment by Geokureli** [comment]
Lots of questions:
what command are you running?
is the build successful?
why is your game inside flixel-tools?
does your game have a Project.xml in the HaxeGame folder?

**Reply by NikRetaNCAM** [reply]
> Lots of questions: what command are you running? is the build successful? why is your game inside flixel-tools? does your game have a Project.xml in the HaxeGame folder?

1) lime test windows
2) i can launch the game through windows explorer, so i guess yeah
3) it wasn't working correctly if it was just on D
4) yeah ofc

**Comment by RichardBray** [comment]
Have you been able to do a successful html5 build?
